### PR TITLE
PR for History management

### PR DESCRIPTION
**Pull request by SK02K1**
# History management added

## Features
- Add video to history when the single video page is mounted.
- Show remove from history control on the video details card if that particular video is present in the history.
- Remove individual videos from history.
- Remove all videos from history using ```Clear history``` CTA on the history page.

Screenshot 2022-03-30 at 9 51 33 PM

Screenshot 2022-03-30 at 9 51 45 PM

Screenshot 2022-03-30 at 9 52 08 PM

### CODE REVIEW

File: src/components/VideoDetailsCard/VideoDetailsCard.jsx [hunk 1]
```diff
@@ -1,12 +1,22 @@
-import { useLocation, useNavigate } from 'react-router-dom';
 import './VideoDetailsCard.css';
+import { useState } from 'react';
```
Comment (VishakAmin) [inline src/components/VideoDetailsCard/VideoDetailsCard.jsx:2]: Separate local import from module import 
Ex:
```
//All module import 
import { useNavigate } from 'react-router-dom';

//All local imports
import { isAlreadyInHistory } from '../../utils';
```

**Review by lomesshh — APPROVED** [review]
Clean and modular code
UI looks clean
all functionalities are working as expected

looks good to merge

**Review by VishakAmin — COMMENTED** [review]
Looks really good left a comment for readability enchantment

File: src/components/VideoDetailsCard/VideoDetailsCard.jsx (hunk 1 above)
Comment (SK02K1) [reply src/components/VideoDetailsCard/VideoDetailsCard.jsx:2]: @VishakAmin thanks for the suggestion , will surely do that as code enhancement later

**Pull request merged**